modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/MVar.kt

```diff
@@ -288,18 +288,18 @@ interface MVar<F, A> {
     fun <F, A> uncancelableOf(initial: A, AS: Async<F>): Kind<F, MVar<F, A>> =
       UncancelableMVar(initial, AS)
 
-    operator fun <F> invoke(AS: Async<F>) = object : MVarPartialOf<F> {
+    operator fun <F> invoke(AS: Async<F>) = object : MVarFactory<F> {
```

Comment:
👍

Reply:
```suggestion
    fun <F> factory(AS: Async<F>) = object : MVarFactory<F> {
```

Consistent with `Ref`